spec/features/users_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'rails_helper'
+
+RSpec.feature 'Users#index', type: :feature do
+  let(:user) { FactoryBot.create(:user) }
+
+  before do
+    FactoryBot.create(:post, user:)
+    FactoryBot.create_list(:user, 3)
+    visit users_path
+    sleep(5)
+  end
+
+  scenario 'I can see the profile picture for each user' do
+    user.photo = '<span class="photo"> <img src="https://robohash.org/voluptatibusperspiciatissuscipit.png?size=300x300&amp;set=set1" width="100" height="100" /> </span>'
+    within(first('.user-list')) do
+      expect(page.body).to include(user.photo)
+    end
+  end
```

Comment:
- [ ] We have some tests failing. Kindly fix the failing tests and resubmit. Your seed file has issues and rises an error. Since this is a testing milestone, it can be good to remove them or have the issue fixed. In case you don't have the same error, delete the project and clone it from GitHub.

![spec3](https://user-images.githubusercontent.com/53124548/216257987-878fea81-e3b2-40d2-a36a-58d57ef678e8.png)